<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="5" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10343" started-at="2018-08-04T22:00:02Z" finished-at="2018-08-04T22:00:13Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10343" started-at="2018-08-04T22:00:02Z" finished-at="2018-08-04T22:00:13Z">
      <class name="com.cbt.tests.AccountActivityTests">
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="setUpTest" is-config="true" duration-ms="366" started-at="2018-08-04T22:00:02Z" finished-at="2018-08-04T22:00:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="setUp" is-config="true" duration-ms="7617" started-at="2018-08-04T22:00:03Z" finished-at="2018-08-04T22:00:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="LogingIn()[pri:0, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="LogingIn" is-config="true" duration-ms="1679" started-at="2018-08-04T22:00:10Z" finished-at="2018-08-04T22:00:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogingIn -->
        <test-method status="FAIL" signature="defaultSavings()[pri:1, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="defaultSavings" duration-ms="12" started-at="2018-08-04T22:00:12Z" description="Verify the default Account type" finished-at="2018-08-04T22:00:12Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.cbt.tests.AccountActivityTests.defaultSavings(AccountActivityTests.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- defaultSavings -->
        <test-method status="FAIL" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="tearDown" is-config="true" duration-ms="621" started-at="2018-08-04T22:00:12Z" finished-at="2018-08-04T22:00:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=defaultSavings status=FAILURE method=AccountActivityTests.defaultSavings()[pri:1, instance:com.cbt.tests.AccountActivityTests@5622fdf] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.cbt.tests.TestBase.tearDown(TestBase.java:68)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.cbt.tests.AccountActivityTests@5622fdf]" name="tearDownTest" is-config="true" duration-ms="5" started-at="2018-08-04T22:00:13Z" finished-at="2018-08-04T22:00:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- com.cbt.tests.AccountActivityTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
=======
<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Smoke tests" duration-ms="13042" started-at="2018-08-04T04:19:56Z" finished-at="2018-08-04T04:20:09Z">
    <groups>
    </groups>
    <test name="My Money Map tests" duration-ms="13042" started-at="2018-08-04T04:19:56Z" finished-at="2018-08-04T04:20:09Z">
      <class name="com.zerob.login.LoginTests">
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.zerob.login.LoginTests@91161c7]" name="setUpTest" is-config="true" duration-ms="294" started-at="2018-08-04T04:19:56Z" finished-at="2018-08-04T04:19:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.zerob.login.LoginTests@91161c7]" name="setUp" is-config="true" duration-ms="11008" started-at="2018-08-04T04:19:56Z" finished-at="2018-08-04T04:20:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.zerob.login.LoginTests@91161c7]" name="loginTest" duration-ms="1255" started-at="2018-08-04T04:20:07Z" finished-at="2018-08-04T04:20:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.zerob.login.LoginTests@91161c7]" name="tearDown" is-config="true" duration-ms="4" started-at="2018-08-04T04:20:08Z" finished-at="2018-08-04T04:20:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTest status=SUCCESS method=LoginTests.loginTest()[pri:0, instance:com.zerob.login.LoginTests@91161c7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.zerob.login.LoginTests@91161c7]" name="tearDownTest" is-config="true" duration-ms="453" started-at="2018-08-04T04:20:08Z" finished-at="2018-08-04T04:20:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- com.zerob.login.LoginTests -->
    </test> <!-- My Money Map tests -->
  </suite> <!-- Smoke tests -->
</testng-results>
>>>>>>> f95be553a1b2f166dca24bc50cdb7e4e4a8c875e
